features/step_definitions/portfolio.rb

```diff
@@ -110,6 +110,23 @@ def getUUID(itemName)
   click_on item
 end
 
+
+Quand("on met {string} dans la barre de recherche et on clique sur {string}") do |value1, value2|
+    #fill_in "Rechercher", with: 'value1', visible: false
+
+    #within("form-control#Rechercher") do
+    #fill_in "Rechercher", :with => @visitor[:value2]
+    find('#Rechercher', visible: true).set value1
+    find('#link', visible: true).click
+    #click_on(value2)
+#end
+
+end
+
+Alors("le dessin {string} est affiché") do |valeur|
```

Comment:
Please use generic term instead of domain-specific term (`item` instead of `dessin`).
Reuse the already existing step (see [code](https://github.com/Hypertopic/Porphyry/blob/v7/features/step_definitions/portfolio.rb#L141-L143)).